src/main/kotlin/verticles/RESTVerticle.kt

```diff
@@ -200,8 +192,53 @@ class RESTVerticle(
 
     }
 
+    private fun parseContainedElements(elementId: String, data: JsonObject, elementIDToParentPath: Map<String, String>): List<Pair<String, String>> {
+
+        val element = safeReadKey(elementId, data, data::getJsonObject)
+        val elementStatus = safeReadKey("status", element, element!!::getInteger)
```

Comment:
An exception can be thrown here, which is not caught, because of the not-null assertion operator.